demos/instantsearchjs/ecommerce/src/app.css

```diff
@@ -431,7 +433,7 @@ footer {
 }
```

Comment:
Can't comment above, but we might want to add a `margin-bottom: 24px;` to `#pagination:empty` (l. 420) too so there is a space between the pagination and footer, WDYT?

![image](https://github.com/algolia/doc-onboarding/assets/662153/dc8a883f-7090-400c-8727-5f027729e6a7)